### PR TITLE
mint an nft

### PR DESCRIPTION
**Pull request by jjasonlan**
Mints an NFT for later use.

nft here: https://testnets.opensea.io/collection/capsuleearlyadopter/overview

https://github.com/capsule-org/react-example/assets/12842310/4d895739-651d-4139-a846-b1d4431e4a8e

### CODE REVIEW

**Review by nsquare3 — COMMENTED** [review]
looks good! do you have the PR for user-management where you increased the faucet amount? also make sure to get Nitya's signoff on a demo of the product flow

**Comment by jjasonlan** [comment]
> looks good! do you have the PR for user-management where you increased the faucet amount? also make sure to get Nitya's signoff on a demo of the product flow

Didn't end up needing to increase the faucet amount!

File: src/nft-marketplace/components/nft/NFT.jsx [hunk 1]
```diff
@@ -231,10 +241,18 @@ const NFT = ({ environment, capsule }) => {
...
+                          await sendTx().then((isSuccess) => {
+                            if (isSuccess) {
+                              setTxState("sent");
+                            } else {
+                              setTxState("error");
+                            }
+                          }).catch(e => {
+                            console.error(e);
+                            setTxState("error");
+                          })
```
Comment (nsquare3) [inline src/nft-marketplace/components/nft/NFT.jsx]: can we refactor to:
```
try {
  const isSuccess = await sendTx();
  if (isSuccess) {
    setTxState("sent");
  } else {
     setTxState("error");
  }
} catch (e) {
  console.error(e);
  setTxState("error");
}
```

**Review by nsquare3 — COMMENTED** [review]
looks great! just one comment

**Review by nsquare3 — APPROVED** [review]

**Pull request merged**